Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -61,21 +61,18 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 PercExcl = NaN;
 MinimumtValue = NaN;
 
-if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTE
+if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTE || x.modules.asl.bContainsDeltaM
     % ENABLE is disabled if multiPLD/TE

```

Comment:
Change into:
`% ENABLE is temporary disabled if multiPLD/TE.`

Reply:
2ac98fc